### PR TITLE
Cockpit 278

### PR DESCRIPTION
**Pull request by jelly**
https://jelly.github.io/cockpit-project.github.io/blog/cockpit-278.html

### CODE REVIEW

**Comment by jelly** [comment]
Waiting on gating of cockpit to update the bodhi links.

File: _posts/2022-10-18-cockpit-278.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Metrics: Display individual disk read/write usage
+
+Metrics page now supports monitoring throughput of each individual disk when multiple disks are detected.
+![screenshot of display individual disk read/write usage]({{ site.baseurl }}/images/278-metrics-display-individual-disk-read-write-usage.png)
+
+
+## Pod CPU, memory, port and volume details
+
+Podman now shows the CPU and memory usage of all the containers in a pod, the CPU usage shown is the highest CPU usage of one of the containers in the pod. Next to the CPU and memory information the exposed ports and volumes are shown, clicking on the port icon shows the port mapping of the pod.
```
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2022-10-18-cockpit-278.md:25]: I think "A pod's CPU usage is the highest CPU usage of one of the containers in the pod." is probably not needed? It's kind of an implementation detail, right?

File: _posts/2022-10-18-cockpit-278.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+![screenshot of ## pod cpu, memory, port and volume details]({{ site.baseurl }}/images/278-pod-cpu-memory-port-and-volume-details.png)
+
```
Comment (garrett) [inline _posts/2022-10-18-cockpit-278.md:28]:
```suggestion
Clicking on the port icon shows the port mapping of the pod.
```

Moving this part be more contextual with the screenshot.

Hopefully GitHub does the right thing with regard to spacing in the file. If not, we can easily fix it.

File: _posts/2022-10-18-cockpit-278.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+![screenshot of ## pod cpu, memory, port and volume details]({{ site.baseurl }}/images/278-pod-cpu-memory-port-and-volume-details-2.png)
+
+## Create new pod support
+
+Podman now allows the creation of pods with an optional port or volume mapping.
```
Comment (garrett) [inline _posts/2022-10-18-cockpit-278.md:34]:
```suggestion
Podman can now create pods, including optional port and volume mappings.
```

File: _posts/2022-10-18-cockpit-278.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+![screenshot of ## ability to create new pod group]({{ site.baseurl }}/images/278-ability-to-create-new-pod-group.png)
+
+
+![Screenshot from 2022-10-04 11-53-51]({{ site.baseurl }}/images/278-ability-to-create-new-pod-group-2.png)
```
Comment (garrett) [inline _posts/2022-10-18-cockpit-278.md:39]: I have an update to this screenshot, to make the framing more balanced. I'll modify it in the PR.

**Review by garrett — CHANGES_REQUESTED** [review]
I've made a bunch of changes to the text. Screenshot update is pending and I'll update it in a moment.

File: _posts/2022-10-18-cockpit-278.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+## Try it out
+
+Cockpit 278 and cockpit-podman 55 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/278)
+* [Cockpit Fedora 37]()
+* [Cockpit Fedora 36]()
```
Comment (garrett) [inline _posts/2022-10-18-cockpit-278.md:50]:
```suggestion
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-614e5cb4e7)
```

**Pull request merged**